kiali-operator/templates/clusterrole.yaml

```
@@ -184,7 +185,7 @@ rules:
   - watch
 # The permissions below are for Kiali itself; operator needs these so it can escalate when creating Kiali's roles
 - apiGroups: [""]
-  resources:
+  resources:  
```

this is a no-op change (spaces at EOL?) - revert
```suggestion
  resources:
```